src/quo/components/tags/context_tag/view.cljs

```diff
@@ -94,92 +96,106 @@
-                                            :blur?               blur?
-                                            :customization-color customization-color})
-     :accessibility-label :context-tag}
-    (case type
-      :default
-      [tag-skeleton {:theme theme :size size :text full-name}
-       [user-avatar/user-avatar
-        {:full-name           full-name
-         :profile-picture     profile-picture
-         :size                (if (= size 24) :xxs 28)
-         :status-indicator?   false
-         :ring?               false
-         :customization-color customization-color}]]
-
-      :multiuser
-      [preview-list/view {:type :user :size :size-20}
-       users]
-
-      :multinetwork
-      [preview-list/view {:type :network :size :size-20}
-       networks]
-
-      :audio
-      [tag-skeleton {:theme theme :text (str duration)}
-       [rn/view {:style (style/audio-tag-icon-container customization-color theme)}
-        [icons/icon :i/play {:color style/audio-tag-icon-color :size 12}]]]
-
-      :group
-      [tag-skeleton {:theme theme :size size :text group-name}
-       [group-avatar/view
-        {:icon-name           :i/members
-         :size                (if (= size 24) :size-20 :size-28)
-         :customization-color (colors/custom-color customization-color 50)}]]
-
-      (:channel :community)
-      [communities-tag (assoc props :channel? (= type :channel))]
-
-      :token
-      [tag-skeleton {:theme theme :size size :text (str amount " " token)}
-       [token/view
-        {:style (style/token-logo size)
-         :token token
-         :size  (if (= size 24) :size-20 :size-28)}]]
-
-      :network
-      [tag-skeleton {:theme theme :size size :text network-name}
-       [rn/image {:style (style/circle-logo size) :source network-logo}]]
-
-      :collectible
-      [tag-skeleton
-       {:theme       theme
-        :size        size
-        :text        (str collectible-name " #" collectible-number)
-        :shrinkable? true}
-       [rn/image {:style (style/rounded-logo size) :source collectible}]]
-
-      :account
-      [tag-skeleton {:theme theme :size size :text account-name}
-       [account-avatar/view
-        {:customization-color customization-color
-         :emoji               emoji
-         :size                (if (= size 24) 20 28)}]]
-
-      :address
-      [address-tag props]
-
-      :icon
-      [icon-tag props]
-
-      :wallet-user
-      [tag-skeleton {:theme theme :size size :text full-name}
-       [wallet-user-avatar/wallet-user-avatar
-        {:full-name           full-name
-         :size                (if (= size 24) :size-20 :size-24)
-         :customization-color customization-color}]]
-
-      :dapp
-      [tag-skeleton {:theme theme :size size :text dapp-name}
-       [rn/image {:style (style/circle-logo size) :source dapp-logo}]]
-
-      nil)]])
+  (let [[image-error? set-image-error] (rn/use-state false)]
```

Comment:
This has been added because some collectibles have links to broken images, if so, we try to display these images and we end up showing the placeholder if the images contain errors